_includes/openring_out.html

```diff
@@ -1,74 +0,0 @@
-<section class="webring">
```

Comment:
Why is this file here? We don't have openring_out.html in the master, only openring_template_out.html. I think this is something local to your branch.

Reply:
Actually we have this file in _includes/openring_out.html in the master, I removed it because we probably won't use it again.

![image](https://user-images.githubusercontent.com/43859386/159338414-9841d1ab-6626-4941-9b17-6b20ec799ab8.png)